src/Acuminator/Acuminator.Analyzers/StaticAnalysis/LongOperationDelegateClosures/LongOperationDelegateTypeClassifier.cs

```diff
@@ -61,8 +61,11 @@ internal class LongOperationDelegateTypeClassifier
 				case DelegateNames.StartOperation:
 					var longRunDelegate = semanticModel.GetSymbolInfo(methodAccessNode, cancellationToken).Symbol as IMethodSymbol;
 
-					if (longRunDelegate != null && longRunDelegate.IsStatic && pxContext.PXLongOperation.Equals(longRunDelegate.ContainingType))
+					if (longRunDelegate != null && longRunDelegate.IsStatic && longRunDelegate.DeclaredAccessibility == Accessibility.Public &&
```

Comment:
Why to filter only to public?

Reply:
We don't want to check non public API.  There are some internal `StartOperation`  API methods
![image](https://user-images.githubusercontent.com/7687400/161773033-052e5fe5-42c8-46d1-beb7-36af9b6218b8.png)

There is also a probability that new internal methods with this name would be added in the future. It's easier to change the internal code since such changes are not breaking changes